dbms/src/Common/Config/ConfigReloader.cpp

```diff
@@ -73,29 +73,68 @@ void ConfigReloader::run()
     }
 }
 
+bool ConfigReloader::tlsCertUpdated(Poco::AutoPtr<Poco::Util::AbstractConfiguration> config, bool init)
```

Comment:
looks like for `main_config_reloader`, `init` is always false.

Reply:
No. In ctor of ConfigReloader, the `force` is `true`.